operations.go

```diff
@@ -0,0 +1,48 @@
+package oas
+
+import "github.com/go-openapi/spec"
+
+const getSpecificationOperationPath = "/openapi.yaml"
```

Comment:
Why not? It does not contradict with RESTful. And it does not only look like a file - actually it is file :) We can also add `/openapi.json` endpoint to serve spec in json format in the future, if need.

Reply:
> actually it is file 

No, it's not) It's a template and it's different each time. More like dynamic page of any other json/xml/whatever-response.

> We can also add /openapi.json

Or add `/spec?format=json` endpoint?

Reply:
> More like dynamic page of any other json/xml/whatever-response.

So what's wrong with **dynamic** file? :thinking:

> No, it's not) 

see `type: file` in the response schema:
```yaml
  /openapi.yaml:
    get:
      operationId: X-Get-Specification
      produces:
      - application/vnd.oai.openapi;version=2.0
      responses:
        "200":
          description: OpenAPI specification in YAML format
          schema:
            type: file
        "500":
          description: Internal Server Error
      summary: Returns this OpenAPI specification in YAML format.
```

But anyway, I can agree that it can be just `/openapi` with optional query `?format=json`.

---

I've skimmed through many related issues and many repositories to see what people do. And they do different :frowning:

For example, `petstore.swagger.io` serves API under `http://petstore.swagger.io/v2` and spec under `http://petstore.swagger.io/v2/swagger.json`

Some projects, like PostgREST, [serve](https://postgrest.com/en/v4.4/api.html#openapi-support) spec under the base path (in petstore example it would be just  `http://petstore.swagger.io/v2`). This seems reasonable, what do you think of this approach?

--- 

Ref: https://github.com/OAI/OpenAPI-Specification/issues/724

Reply:
> serve spec under the base path

I like it

Reply:
@tetafro done